src/components/VueMoney.js

```diff
@@ -3,7 +3,13 @@ import { checkToOnlyNumber, moneyFormat } from "@utils/moneyUtils";
 export default {
     name: "VueMoney",
     props: {
-        minMax: {
+        value: {
+            required: true,
+            type: [Number, String],
+            default: 0
+        },
+
+        min_max: {
```

Comment:
`min_max, oldValue` are two separate methodologies of naming. Either do `minMax` or `old-value`